workshop_app/views.py

```diff
@@ -195,102 +197,102 @@ def edit_profile(request):
+        instructor=user.id,
+        date__gte=today,
+    ).order_by('-date')
 
-            today = timezone.now()
-            proposed_workshop = Workshop.objects.filter(
-                instructor=user.id,
-                date__gte=today,
-            ).order_by('-date')
+    proposed_workshop_pending = Workshop.objects.filter(
+        status=0
+    ).order_by('-date')
 
-            proposed_workshop_pending = Workshop.objects.filter(
-                status=0
-            ).order_by('-date')
+    workshops = list(proposed_workshop) + list(proposed_workshop_pending)
+    return render(request, 'workshop_app/workshop_status_instructor.html',
+                  {"workshops": workshops,
+                   "today": today})
 
-            workshops = list(proposed_workshop) + list(proposed_workshop_pending)
 
-            return render(request, 'workshop_app/my_workshops.html',
-                          {"workshops": workshops,
-                           "today": today})
+@login_required
+def accept_workshop(request, workshop_id):
+    user = request.user
+    if not is_instructor(user):
+        return redirect(get_landing_page(user))
+    workshop = Workshop.objects.get(id=workshop_id)
+    # Change Status of the selected workshop
+    workshop.status = 1
+    workshop.instructor = user
+    workshop.save()
+
+    coordinator_profile = workshop.coordinator.profile
+
+    # For Instructor
+    send_email(request, call_on='Booking Confirmed',
+               user_position='instructor',
+               workshop_date=str(workshop.date),
+               workshop_title=workshop.workshop_type.name,
+               user_name=str(coordinator_profile.user.get_full_name()),
+               other_email=workshop.coordinator.email,
+               phone_number=coordinator_profile.phone_number,
+               institute=coordinator_profile.institute
+               )
+
+    # For Coordinator
+    send_email(request, call_on='Booking Confirmed',
+               workshop_date=str(workshop.date),
+               workshop_title=workshop.workshop_type.name,
+               other_email=workshop.coordinator.email,
+               phone_number=request.user.profile.phone_number
+               )
+    return redirect(reverse('workshop_status_instructor'))
 
-        # Coordinator view
-        else:
-            workshops = Workshop.objects.filter(
-                coordinator=user.id
-            ).order_by('-date')
-            return render(request, 'workshop_app/my_workshops.html',
-                          {"workshops": workshops})
-    # Not logged in view
-    else:
-        return redirect('/login/')
+
+@login_required
+def change_workshop_date(request, workshop_id):
+    user = request.user
+    if not is_instructor(user):
+        return redirect(get_landing_page(user))
+    if request.method == 'POST':
+        client_data = request.POST
+        new_workshop_date = datetime.strptime(client_data.get('new_date'), "%Y-%m-%d")
```

Comment:
You don't seem to be using `client_data` anywhere else in the `view` except here. Can you do it this way
` new_date = request.POST.get('new_date')` ?